via/views/view_pdf.py

```diff
@@ -16,14 +16,18 @@
     # We have to keep the leash short here for caching so we can pick up new
     # immutable assets when they are deployed
     http_cache=0,
-    decorator=(checkmate_block, has_secure_url_token),
+    decorator=(has_secure_url_token,),
 )
 def view_pdf(context, request):
     """HTML page with client and the PDF embedded."""
 
+    url = context.url()
+
+    request.checkmate.raise_if_blocked(url)
```

Comment:
Out of scope for this PR but I wonder what's the pyramid-y way of annotating the request  on `route_by_content` (or directly on `ViaCheckmateClient`) and skip a second check if it happens

Reply:
Yeah. In our initial solution to this we put a signed token on that say'd we checked it, but it caused some issue I can't remember and instead of solving it, we removed it.

I think our approach right now is:

 * Checkmate is super fast
 * It's better to check twice and not have an error
 * We can optimise later (or not) if we need to